docs/Developers/Writing_Native_Bears.rst

```diff
@@ -411,6 +411,8 @@ strings as a value:
     class SomeBear(Bear):
         LANGUAGES = {'C', 'CPP','C#', 'D'}
 
+.. include:: /home/imrekcah/coala/docs/Developers/language_naming_format.rst
```

Comment:
Same here.

Reply:
@abhishalya I did not get you

Reply:
@abhishalya Done brother

Reply:
@budukhyash Sorry I think you missed the previous comment. The location here does not exist, how do you think it will work.
Basically, the tests need to pass.